libraries/AP_RangeFinder/AP_RangeFinder_LightWareSerial.cpp

```diff
@@ -121,10 +124,24 @@ bool AP_RangeFinder_LightWareSerial::get_reading(float &reading_m)
 
     // all readings were invalid so return out-of-range-high value
     if (invalid_count > 0) {
-        reading_m = MIN(MAX(LIGHTWARE_DIST_MAX_CM, max_distance_cm() + LIGHTWARE_OUT_OF_RANGE_ADD_CM), UINT16_MAX) * 0.01f;
+        reading_m = MIN(MAX(LIGHTWARE_DIST_MAX_CM, distance_cm_max + LIGHTWARE_OUT_OF_RANGE_ADD_CM), UINT16_MAX) * 0.01f;
         return true;
     }
 
     // no readings so return false
     return false;
 }
+
+// check to see if distance returned by the LiDAR is a known lost-signal distance flag
+bool AP_RangeFinder_LightWareSerial::is_lost_signal_distance(const float &distance_cm, const int16_t &distance_cm_max)
```

Comment:
I don't think we should bother passing these arguments as const references.  They're only 4 bytes and 2 bytes.. let's just pass them as regular (non-const) float and int16_t.

Reply:
@rmackay9 Latest change does as you suggested and doesn't bother passing arguments into that method as const references--they're now just simply passed by value. Also, squashed my commits into a single commit. Thank you.